src/main/resources/api/service_openapi.yaml

```diff
@@ -862,6 +862,24 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/gcp/bqdatasets/{resourceId}:
```

Comment:
nit: we should probably be consistent on if we're '-' separating words in our path. `bqdatasets` vs `ai-notebook-instances`

Reply:
I think we've generally used no separator. elsewhere, we have `cloudcontexts` and `bigquerydatasets` (which is a separate issue, I need to make controlled BQ datasets consistent with referenced BQ datasets).
I see `-` separators in a couple async result endpoints though, PR-726 to fix since that will (potentially) be a breaking API change.